<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorConfigFont</class>
 <widget class="QWidget" name="EditorConfigFont">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="defaultFontselectLabel">
       <property name="text">
        <string>Default font</string>
       </property>
       <property name="buddy">
        <cstring>defaultFontselect</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="defaultFontselect"/>
     </item>
     <item>
      <widget class="QSpinBox" name="defaultFontsize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="monospaceGroupBox">
     <property name="title">
      <string>Monospace font options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="monospaceFontselectLabel">
        <property name="text">
         <string>Monospace font</string>
        </property>
        <property name="buddy">
         <cstring>monospaceFontselect</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFontComboBox" name="monospaceFontselect"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="monospaceFontsizeapply">
        <property name="text">
         <string>Font size follows formatting</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="monospaceFontsize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="codeGroupBox">
     <property name="title">
      <string>Code font options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="codeFontselectLabel">
          <property name="text">
           <string>Code font</string>
          </property>
          <property name="buddy">
           <cstring>codeFontselect</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="codeFontselect"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="codeSelectColorLabel">
          <property name="text">
           <string>Code color</string>
          </property>
          <property name="buddy">
           <cstring>codeSelectColorButton</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="codeSelectColorButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="codeFontsizeapply">
          <property name="text">
           <string>Font size follows formatting</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSpinBox" name="codeFontsize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>250</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="codeIndentsizeapply">
          <property name="text">
           <string>Indent size follows formatting</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSpinBox" name="codeIndentsize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>250</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultFontselect</tabstop>
  <tabstop>defaultFontsize</tabstop>
  <tabstop>monospaceFontselect</tabstop>
  <tabstop>monospaceFontsizeapply</tabstop>
  <tabstop>monospaceFontsize</tabstop>
  <tabstop>codeFontselect</tabstop>
  <tabstop>codeSelectColorButton</tabstop>
  <tabstop>codeFontsizeapply</tabstop>
  <tabstop>codeFontsize</tabstop>
  <tabstop>codeIndentsizeapply</tabstop>
  <tabstop>codeIndentsize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
